regolith/helpers/l_progressreporthelper.py

```diff
@@ -28,10 +28,10 @@ def subparser(subpi):
                        help="Filter projecta for this project lead"
                        )
     subpi.add_argument("-s", "--stati", nargs="+",
-                       choices=ALLOWED_STATI,
+                       choices=PROJECTUM_STATI,
                        help=f"Filter projecta for these stati."
-                            f" Default is all projecta",
-                       default=None,
+                            f" Default is {PROJECTUM_ACTIVE_STATI+PROJECTUM_FINISHED_STATI}",
```

Comment:
I think this won't print in a very pretty way, you have to unpack the list.  Prepend a * and put a comma at the end to make the unpacking work.